frontend/src/api/questions.ts

```diff
@@ -8,11 +8,20 @@ export async function create(data: QuestionPostData): Promise<AxiosResponse<Ques
   return questionClientApi.post(`${URL}`, data);
 }
 
+export async function get(data?: String): Promise<AxiosResponse<any>> {
+  if (data) {
+    return questionClientApi.get(`${URL}/${data}`);
+  }
+  else {
+    return questionClientApi.get(`${URL}`);
+  }
+}
+
 export async function update(data: QuestionPatchData): Promise<AxiosResponse<Question>> {
   return questionClientApi.patch(`${URL}/${data._id}`, data);
 }
 
 export async function remove(data: QuestionDeleteData): Promise<AxiosResponse<Question>> {
-  return questionClientApi.delete(`${URL}/delete`, { data });
+  return questionClientApi.delete(`${URL}/${data._id}`, { data });
```

Comment:
```suggestion
  return questionClientApi.delete(`${URL}/${data._id}`);
```
Think can remove the data portion since question service doesnt use it.